www/src/pages/components/forms.js

```diff
@@ -292,6 +294,16 @@ export default withLayout(function FormControlsSection({ data }) {
       <h3>Inline</h3>
       <ReactPlayground codeText={CheckCustomInline} />
 
+      <LinkedHeading h="3" id="forms-custom-file">
+        File
+      </LinkedHeading>
+      <p>A custom styled File uploader.</p>
+
+      <ReactPlayground codeText={File} />
```

Comment:
ultimately want to add a more about how to customize the inner markup like https://react-bootstrap.github.io/components/forms/#forms-check-api

Reply:
This has been fixed and committed.